### DISCUSSION TITLE
Questions about the output of beet update

### DISCUSSION THREAD

**Discussion by ovizii** [beetbox/beets#5183] [Q&A] [answered]
I had edited some files outside of beets and wanted to get beet up-to-date so I ran a beet update. 

I'm wondering how to read the cryptic output of this command. Here are some samples:

```
U2 - Achtung Baby - “Baby” Acrobat
  albumtype: [ -> a
  albumtypes: ['[', "'", 'a', "'", ',', ' ', "'", 'l', "'", ',', ' ', "'", 'b', "'", ',', ' ', "'", 'u', "'", ',', ' ', "'", 'm', "'", ']'] -> ['a', 'l', 'b', 'u', 'm']


U2 & Johnny Cash - Ring of Fire: The Legend of Johnny Cash - The Wanderer
  albumtype: a -> compilation
  albumtypes: ['a', 'l', 'b', 'u', 'm', ';', ' ', 'c', 'o', 'm', 'p', 'i', 'l', 'a', 't', 'i', 'o', 'n'] -> ['compilation']
```

**Comment by pyyhttu** [comment, marked as answer]
Looks like #4528?

**Reply by ovizii** [reply]
Thanks! I searched but wasn't aware of the right keywords. I will close this issue as the one you linked has all the details I need to fix this mess. I read that a new beets version is about to be released, so I can wait a bit to fix this.

Also see: https://github.com/beetbox/beets/pull/4582#issuecomment-1445023493 for manually checking and fixing the issue.
